src/components/Dropdown/Dropdown.tsx

```diff
@@ -61,6 +66,7 @@ export class Dropdown extends React.Component<DropdownProps, State> {
     placement: 'bottom',
     mobileNativeSelect: false,
     upgrade: false,
+    optionsContainerId: 'dropdown-options-container',
```

Comment:
Why no call this `id` 🤔 ?

Reply:
it's a very specific id for the options container who is generated inside ui-core. You can check how here : https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/dropdown-content/DropdownContent.tsx#L217 and https://github.com/wix/wix-ui/blob/master/packages/wix-ui-core/src/components/dropdown-content/DropdownContent.tsx#L150